flixel/input/mouse/FlxMouseEventManager.hx

```diff
@@ -36,10 +36,20 @@ class FlxMouseEventManager extends FlxBasic
 {
 	private static var _registeredObjects:Array<ObjectMouseData<FlxObject>> = [];
 	private static var _mouseOverObjects:Array<ObjectMouseData<FlxObject>> = [];
+	private static var _mouseDownObjects:Array<ObjectMouseData<FlxObject>> = [];
+	private static var _mouseClickedObjects:Array<ObjectMouseData<FlxObject>> = [];
+	
+	private static var _mouseClickedTime:Int = -1;
 
 	private static var _point:FlxPoint = FlxPoint.get();
 	
 	/**
+	 * The maximum amount of time between two clicks that is considered a double click, in milliseconds.
+	 * @since 4.4.0
+	 */
+	public static var MAX_DOUBLE_CLICK_DELAY:Int = 500;
```

Comment:
Hm.. the `UPPER_CASE` naming implies to me that this is a constant (declared `inline` for instance) that can't be changed.